ropgadget/loaders/elf.py

```diff
@@ -26,6 +26,7 @@ class ELFFlags(object):
     EM_PowerPC  = 0x14
     EM_PPC64    = 0x15
     EM_ARM64    = 0xb7
+    EM_RISCV    = 0Xf3
```

Comment:
There is also [raw](https://github.com/JonathanSalwan/ROPgadget/blob/master/ropgadget/loaders/raw.py#L35) format loader.

Reply:
added support for riscv 64b | compressed. will not add 32bit yet